bigquerydb/client.go

```diff
@@ -82,10 +83,16 @@ func NewClient(logger log.Logger, googleAPIjsonkeypath, googleAPIdatasetID, goog
 		timeout:   remoteTimeout,
 		ignoredSamples: prometheus.NewCounter(
 			prometheus.CounterOpts{
-				Name: "prometheus_bigquery_ignored_samples_total",
+				Name: "storage_bigquery_ignored_samples_total",
 				Help: "The total number of samples not sent to BigQuery due to unsupported float values (Inf, -Inf, NaN).",
 			},
 		),
+		recordsFetched: prometheus.NewCounter(
+			prometheus.CounterOpts{
+				Name: "storage_bigquery_records_fetched",
+				Help: "Total number of records fetched in a batch.",
```

Comment:
I don't think this is per batch. Isn't this the "Total number of records fetched", which start with 0 at the start of the process?

Reply:
corrected